code/game/turfs/closed.dm

```diff
@@ -429,6 +430,7 @@
 	icon = 'icons/turf/dropship.dmi'
 	icon_state = "1"
 	plane = GAME_PLANE
+	resistance_flags = RESIST_ALL|PLASMACUTTER_IMMUNE
```

Comment:
```suggestion
	resistance_flags = UNACIDABLE
```
You have made every single mineral wall in the game unbreakable and people are yelling at me over it. Please fix this

Reply:
Nevermind, I think I'm being stupid here. Although, plasmacutters already check for RESIST_ALL so you shouldn't need to separately define PLASMACUTTER_IMMUNE